app/assets/javascripts/client/views/directives/inventories/manage_participants.html

```diff
@@ -0,0 +1,14 @@
+<div class="row manage manage-participants plus-button" ng-click="showAddParticipants()">
```

Comment:
Odd that this file is showing up now in this PR when it was last added into the repository...